app/models/star.rb

```diff
@@ -5,7 +5,7 @@ class Star < ApplicationRecord
   belongs_to :project
   after_create_commit :notify_recipient
   before_destroy :cleanup_notification
-  has_many :notifications, as: :notifiable, dependent: :destroy
+  has_many :notifications, as: :notifiable  # rubocop:disable Rails/HasManyOrHasOneDependent
```

Comment:
Why are we removing this?

Reply:
@tachyons 

This is the `activity_notification` gem association for which we do not have a `Notification` model, after running `data_migration` task, we are gonna delete this as well!

`has_noticed_notifications` contain `destroy` as true by default for `noticed_notifications`!

WDYT?